core/templates/components/skill-selector/select-skill-modal.template.html

```diff
@@ -5,13 +5,14 @@ <h3>
   <i class="fa fa-times modal-close-button" ng-click="cancel()"></i>
 </div>
 <div class="modal-body">
-  <select-skill sorted-skill-summaries="skillSummaries"
-                selected-skill-id="selectedSkillId"
-                count-of-skills-to-prioritize="countOfSkillsToPrioritize"
-                categorized-skills="categorizedSkills"
-                untriaged-skill-summaries="untriagedSkillSummaries"
-                allow-skills-from-other-topics="allowSkillsFromOtherTopics">
-  </select-skill>
+  <skill-selector [sorted-skill-summaries]="skillSummaries"
```

Comment:
i think it should be `oppia-skill-selector`, but verify this from maintainers first.

Reply:
@kevintab95 WDYT? Both seem fine to me

Reply:
Maintainers wants that.